db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
Not a typo. When trying to rollback a migration which had a simple `drop_table` statement, I had this error:

```
To avoid mistakes, drop_table is only reversible if given options or a block (can be empty).
```

Consequently, I decided to add an empty block (choosing one of the 2 options) and that made the said migration reversible. And that was the goal of that commit, because I think there is value in having all migrations reversible, whenever possible.

So @benichu, do you still want me to remove those empty blocks?